backend/config/settings/production.py

```diff
@@ -222,3 +222,8 @@
 )
 
 DJANGO_REST_MULTITOKENAUTH_RESET_TOKEN_EXPIRY_TIME = 24 * 7  # 7 days
+
+# URLs for the front-end
+API_URL = env("VITE_BACKEND_URL", default="https://app.coord.dev")
```

Comment:
Could we instead define the defaults in base.py or will that break the dev setup?